src/modules/netinstall/NetInstallPage.cpp

```diff
@@ -0,0 +1,139 @@
+#include "NetInstallPage.h"
+
+#include "./widgets/groupselectionwidget.h"
```

Comment:
FYI, this is not just a consistency thing. Those leading dots can actually break RPM builds (though in this case, probably not, because the path is actually "./widgets" and not just "."). They end up in `#line` directives to the assembler, from there in the debugging sections of the object file, and then you end up with a path of "." in the path table, which `debugedit` wants to canonicalize to the empty string "", but the hack `debugedit` uses to shrink strings breaks down if the string shrinks by exactly 1 character, so it fails with an error.